css/css-flexbox/reference/flex-aspect-ratio-img-row-014-expected.xht

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
`-expected` is common in WebKit, but not really in WPT.

BTW, why are you using XHTML here instead of just HTML with '<!DOCTYPE html>` like in the test?

Reply:
Hmm just because I use some other as reference to create the expectation.